tests/conftest.py

```diff
@@ -7,6 +7,24 @@
 
 import tubular.base as base
 
+"""
+How To Use This Testing Framework
```

Comment:
P: think this description is very good and should hopefully make the whole process clearer :)

Reply:
thanks, hope so!